src/stores/AutoRageshakeStore.ts

```diff
@@ -109,14 +115,19 @@ export default class AutoRageshakeStore extends AsyncStoreWithClient<IState> {
 
     private async onDeviceMessage(ev: MatrixEvent): Promise<void> {
         if (ev.getType() !== AUTO_RS_REQUEST) return;
+        const messageContent = ev.getContent();
+        const recipientRageshake = messageContent["recipient_rageshake"] || "";
         const now = new Date().getTime();
         if (now - this.state.lastRageshakeTime > RAGESHAKE_INTERVAL) {
             await this.updateState({ lastRageshakeTime: now });
             await sendBugReport(SdkConfig.get().bug_report_endpoint_url, {
-                userText: "Auto-reporting decryption error (sender)",
+                userText: `Auto-reporting decryption error (sender) ${recipientRageshake}`,
```

Comment:
It's definitely nasty, but this is what vector-im/element-android#4325 does too, it's the only way we've found to get github to actually linkify the URL so the recipient rageshake will have a "mentioned in issue #..." in its thread.

Reply:
I've been chatting about making changes to the rageshake server to add a `related_issues` field or something, where URLs will actually be links in the issue body; once that happens, we can update both element-web and element-android's autorageshake code to work with that. For now, think it's reasonable to close this thread and get things merged?

Reply:
github should be linkifying any links in the issue body anyway though..?

Reply:
It doesn't, the rageshake bot puts all the form fields in `code quotes` so nothing inside gets parsed as markdown or linkified.  I'm not sure what the reasoning is there; maybe to avoid accidentally tagging people?

Reply:
You are right, could we get a line break before the link just so it doesn't end up in the title at least

Reply:
Great idea, thank you! @BillCarsonFr element-android should probably do the same.